packages/feedback-spa/spaship.yaml

```diff
@@ -1,6 +0,0 @@
-# SPAship config file
```

Comment:
Add the spaship.yaml file back. This needs to be in source control to ensure consistent deployments.

Reply:
Included it back